tests/corvus_test.c

```diff
@@ -83,7 +84,9 @@ int main(int argc, const char *argv[])
     build_contexts();
     struct context *contexts = get_contexts();
 
-    memcpy(&config.node, &conf, sizeof(config.node));
+    config.node = cv_malloc(sizeof(struct node_conf));
```

Comment:
Have you used `valgrind` to check whether there is memory leak in test program?
Please add more complete tests.